### DISCUSSION TITLE
Re-ordering ui elements

### DISCUSSION THREAD

**Discussion by Y4cine** [zauberzeug/nicegui#1840] [Q&A] [answered]
### Question

Say I have the following code (not working as expected)
```py
from nicegui import ui

ui_elements = [ui.button("Button " + str(i), on_click=lambda i = i: move_element(i)) for i in range(10)]

def move_element(n):
    if 0 <= n < len(ui_elements) - 1:
        ui_elements[n], ui_elements[n + 1] = ui_elements[n + 1], ui_elements[n]
    print(n)

with ui.row():
    ui_elements

ui.run()
```
What do I need to do to make this code running, i.e. change the position of ui elements?

**Comment by falkoschindler** [comment, marked as answer]
I'm not sure what the code should be doing exactly (swap element n with its successor?), but here is the documentation about moving elements to a different position: https://nicegui.io/documentation/element#move_elements

**Reply by Y4cine** [reply]
Spent the day searching for this info. Thank you so much.

**Reply by rodja** [reply]
We want to make all infos available as simple as possible. Have you been aware of the search feature on htts://nicegui.io, @Y4cine?

![Screenshot 2023-10-20 at 04 57 17](https://github.com/zauberzeug/nicegui/assets/131391/9ea3648a-e7cc-4096-a4d1-f9ece14079ff)

**Reply by Y4cine** [reply]
I was indeed aware of the search feature and used it already, but for this search I relied on Google.
